localization-proofreading-english/StringTableEnExaminations.xml

```diff
@@ -101,76 +101,76 @@
             <GameDBLocalizedString>      <LocID>EXM_ORAL_CAVITY_INSPECTION</LocID>                              <Text>Oral cavity inspection</Text>  </GameDBLocalizedString>
             <GameDBLocalizedString>      <LocID>EXM_ORAL_CAVITY_INSPECTION_DESCRIPTION</LocID>                  <Text>Examination and visual inspection of the patient's oral cavity. \n\nRequired room: Any office / Diagnostic unit </Text>  </GameDBLocalizedString>
             <GameDBLocalizedString>      <LocID>EXM_PCR_SAMPLING</LocID>                                        <Text>PCR - sampling</Text>  </GameDBLocalizedString>
```

Comment:
Jeg er usikker på linje 103-106. Den ene er PCR sampling og den anden er PCR testing - De har på nuværende tidspunkt samme beskrivelse, men såfremt det er to forskellige ting ift. gameplay, bør de måske uddybes nærmere, således at man pointerer at den ene er sampling af biologisk materiale, og den anden er en laboratorieundersøgelse af det biologiske materiale? - Det samme gør sig egentlig gældende for peritoneal fluid. Jeg har givet bud på ændringer i begge tilfælde.